### DISCUSSION TITLE
叶绿体基因组组装无法成环

### DISCUSSION THREAD

**Discussion by doubleHwithT** [Kinggerm/GetOrganelle#358] [Q&A]
金老师您好!
我在叶绿体组装过程中出现了结果不成环的问题。
我按照Q&A中的参数建议调整了相关参数，减少了-w 增加了-R 同时设置了--reduce-reads-for-coverage=1000，同时添加了-s参考序列（参考序列是去年用GetOrganelle装的，那时候非常顺利，结果也经过了检查，但是我已经找不到去年的参数和log文件了）
调整后的参数命令为:get_organelle_from_reads.py -1 PAK9_clean_R1.fq.gz -2 PAK9_clean_R2.fq.gz -o PAK_PAK9_output -R 20 -k 21,47,67,87,107,127,147 -t 20 -w 90 --reduce-reads-for-coverage 1000 -F embplant_pt -s reference.fasta

做了不同的尝试后，情况虽有所改善，但是依旧会出现组装结果存在两条Scaffold的问题，我现把本次修改参数后运行的log文件和Bandage的图给您查看，请您提出宝贵的建议，非常感谢您！

[PAK9.log.txt](https://github.com/user-attachments/files/17571394/PAK9.log.txt)
![1](https://github.com/user-attachments/assets/11778d40-7a7c-445f-81a0-4c58ba1568e1)
![2](https://github.com/user-attachments/assets/08d53a4c-1773-4286-ad2e-570319f88b3b)

**Comment by 728836672** [comment]
嗨，你好，你的问题得到解决了吗？我遇到了同样的问题，出现了两条脚手架Scaffold。

**Reply by 12hss** [reply]
您好，您的问题有解决了吗，可以给我一些
建议吗

**Comment by JianjunJin** [comment]
首先，--reduce-reads-for-coverage还可以设置得更高一些。

之前成环的跟现在是同一个样吗？不同样本/批次差异可以很大的，测序质量以及基因组本身的特征都会影响成环。如果在同一个类群中总是在同一个地方断开，也可能是这个区域有大量的重复序列。
